src/Carbon/CarbonInterval.php

```diff
@@ -436,6 +436,21 @@ public static function create($years = 1, $months = null, $weeks = null, $days =
         return new static($years, $months, $weeks, $days, $hours, $minutes, $seconds, $microseconds);
     }
 
+    /**
+     * Parse a string into a new CarbonInterval object according to the specified format.
+     *
+     * @param string $format
+     * @param string $interval
+     *
+     * @return void
```

Comment:
`@return static`

This method is not void.

And I suggest adding parameters description and an example (will appear in the API reference of the documentation):
```php
    /**
     * Parse a string into a new CarbonInterval object according to the specified format.
     *
     * @example
     * ```
     * echo Carboninterval::createFromFormat('H:i', '1:30');
     * ```
     *
     * @param string $format  Format of the $interval input string
     * @param string $interval Input string to convert into an interval
     *
     * @return static
     */